main.py

```diff
@@ -13,14 +13,16 @@ def main():
     print('==== starting subselection ====')
 
     ############# INPUTS for pre-processing #############
+    # todo the following should probably be variables that are passed to the function as they seem to be modified
```

Comment:
agreed, this can be targeted in a next release. Relative paths can produce issues on the cluster if the respective job-cwds are not set correctly.

Reply:
What is a job-cwds?

Reply:
Anyway, agree that this can be a future task and doesn't have to be part of this  iteration